sources/api/pluto/src/eks.rs

```diff
@@ -57,6 +46,11 @@ pub(super) async fn get_cluster_network_config(
         .find(|env_var| *env_var != Err(env::VarError::NotPresent))
         .and_then(|s| s.ok());
 
+    let config = aws_config::from_env()
+        .region(Region::new(region.to_owned()))
```

Comment:
I'm a little weirded out by the fact that `aws_sdk_rust` doesn't attempt to do validation on the input string. We also don't do any validation on `settings.aws.regions`. I'm assuming this just takes whatever is given here and generates API endpoints. Just something to keep in mind.

Reply:
We could probably make some guesses surrounding what a region name looked like, but, I feel like we open ourselves to being broken every time a new region is released by doing so.